src/gitDaemon.ts

```diff
@@ -79,6 +80,16 @@ export async function initRepo(
     };
   }
 
+  for (const tag of additionalTags) {
+    await execFilePromise("git", ["commit", "--allow-empty", "-m", `commit for ${tag}`], { cwd: repoPath });
```

Comment:
```suggestion
    await execFilePromise("git", ["-c", "commit.gpgsign=false", "commit", "--allow-empty", "-m", `commit for ${tag}`], { cwd: repoPath });
```

Reply:
I don't know how others have set it up, but I have git configured to sign commits in every repo in a `parity` folder.
With this change it would be easier for me to run the tests.

Reply:
I wouldn't spread git config around all the commands.  

Right now, we set global git config values for CI only: https://github.com/paritytech/gitspiegel/blob/a9c1580c9fc49301993ab9befebd5f5009dba889/src/test/integration/setup/globalSetup.ts#L52-53

But maybe it should 
* be a local config that's created here, in `initRepo()`
* be used for both local runs and CI, to make it independent from devs' configs

WDYT?

Reply:
Good idea.

So if we add to `initRepo()` these:

```
await execFilePromise("git", ["config", "commit.gpgsign", "false"], { cwd: repoPath });
await execFilePromise("git", ["config", "user.email", "bot@example.com"], { cwd: repoPath });
await execFilePromise("git", ["config", "user.name", "Robot Tester"], { cwd: repoPath });
```

We would solve my use case, and we could remove `setUpGitConfig` from CI.